pages/validators/full_node_streaming.md

```diff
@@ -312,6 +316,8 @@ By protocol design, each validator has their own version of the orderbook and op
 
 ![full node streaming diagram](../../artifacts/full_node_streaming_diagram.jpg)
 
+Note that DeliverTx maps to exec mode `execModeFinalize`.
```

Comment:
This is a weird gotcha. Can we fix this on the full node stream side?

Reply:
It's on cosmos sdk end but will think about this later, don't want to block 4.1.4 with this